jest.config.js

```diff
@@ -0,0 +1,177 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
is this ok?

Reply:
Is a config file for jest (with all configurations commented), i think that could be moved to package.json.